### DISCUSSION TITLE
GitHub Action Checks Failing

### DISCUSSION THREAD

**Discussion by slominskir** [389ds/389-ds-base#6102] [Q&A]
Is someone looking into why automated tests never pass?  Can we remove the failing tests from the test suite?   The badge in the README says it all, but it's been that way for at least a month or so now.   Probably goes without saying but main branch probably should have passing tests as steady state.   PR check feedback is annoying and unhelpful as lots of spurious check failed emails having nothing to do with PRs.

**Comment by vashirov** [comment]
We understand your frustration with the current state of the test suite. 
Our team is small and we are currently stretched thin with limited resources. We have been aware of the failing tests and have been looking into the issue, but have not yet been able to dedicate the necessary time and attention to resolve it.

Regarding your suggestion to remove the failing tests from the test suite, we would like to keep them for now as we believe they provide value in catching potential issues. Moreover, there are known issues in the `main` branch highlighted by these failing tests, and removing them would be counterproductive.

**Comment by progier389** [comment]
FYI (to complete @vashirov answer):
- The only test that is know to almost always fails is the WebUI (you can ignore it) but it is the only test we have about the GUI so we prefer not to remove it.
- Any test may also fails randomly (usually 4 or 5 tests fails on each run) because of a dnf race condition issue 
(re running the failed tests usually solves the issue)
- There is indeed a known regression that breaks some tests but it should be fixed quickly by @jchapma with [PR #6104](https://github.com/389ds/389-ds-base/pull/6104)
